readiness-tool/popup.html

```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<!--
+ This page is shown when the extension button is clicked, because the
+ "browser_action" field in manifest.json contains the "default_popup" key with
+ value "popup.html".
+ -->
+<html>
+    <head>
+        <title>Getting Started Extension's Popup</title>
+        <style>
+            html {
+                background-color: #f5f5f5;
+                height: 100px !important;
```

Comment:
Is the height really being set to 100px? I thought it expands with the content of the page...